app/public/stylesheet.css

```diff
@@ -0,0 +1,109 @@
+.banner {
```

Comment:
I had a lot of fun with the styling but still not quite finished - and annoys me that a few elements are misaligned or unstyled in the app. Also think that this file is pretty messing - want to tidy it up a bit.